stable/yugaware/templates/pdb.yaml

```diff
@@ -3,7 +3,7 @@ kind: PodDisruptionBudget
 metadata:
   name: {{ .Release.Name }}-yugaware-pdb
 spec:
-  maxUnavailable: 0
+  maxUnavailable: {{ .Values.yugaware.podDisruptionBudget.maxUnavailable | toJson }}
```

Comment:
Do you need the toJson? Isn't an int value just cleanly applyable to the template?

Reply:
It isn't strictly required but makes errors in the values provide much cleaner errors as the value is restricted to this attribute and doesn't spew out random text that is parsed as yaml.